README.md

```diff
@@ -82,14 +82,29 @@ const client = new ampDynamicContent.ContentClient({
 
 If you need to support old browsers a legacy version of the bundle is provided, however we strongly recommend using a tool like [babel](https://babeljs.io/) in your project to compile the sdk to your exact browser requirements.
 
-### Fetch content
+### Configuration options
+
+| Option             | Description                                                                                             |
+| ------------------ | ------------------------------------------------------------------------------------------------------- |
+| account            | Required (see Note below): Account to retrieve content from Content Delivery v1 API                     |
+| hubName            | Required (see Note below): hubName to retrieve content from Content Delivery v2 API                     |
+| stagingEnvironment | If set, the SDK will request content and media from the staging environment host name specified.        |
+| locale             | If set, the SDK will request content using the locale settings provided.                                |
+| mediaHost          | Allows users with custom hostnames to override the hostname used when constructing media URLs.          |
+| secureMediaHost    | Allows users with custom hostnames to override the hostname used when constructing secure media URLs.   |
+| baseUrl            | Override for the content delivery API base URL                                                          |
+| adaptor            | Allows custom handling of requests which makes testing and supporting non-standard environments easier. |
+
+**Note**: In order to use the client must be configure it with either `account` or `hubName`. If both are supplied the SDK will use the Content Delivery V2 API over Content Delivery V2.
```

Comment:
do you mean it will use Content Delivery 2 over Content Delivery v1?

Reply:
Also, not all users will have CD2 enabled on their hub. Do we need to explicitly call out: _if you have CD2 on your hub, then you can... _ (etc).?

Reply:
I'll correct the typo.

I didn't cover the provisioning aspect of this new delivery API, I thought it was best to leave this out of SDK.

Reply:
No problem - happy to go with the current pattern.